mixer/adapter/kubernetesenv/kubernetesenv.go

```diff
@@ -426,11 +426,8 @@ func initMultiClusterSecretController(b *builder, kubeconfig string, env adapter
 		return fmt.Errorf("could not create K8s client: %v", err)
 	}
 
-	err = secretcontroller.StartSecretController(kubeClient, b.createCacheController,
+	_ = secretcontroller.StartSecretController(kubeClient, b.createCacheController,
```

Comment:
i think getting the controller back here would have been useful for previous issues with controllers, etc. Not sure it matters any more however.